guides/build-api-plugin.md

```diff
@@ -225,6 +225,28 @@ query {
 
 We’ve done everything we need to add volume data to our backpacks and query that information across our store. That’s useful to Leader of the Pack’s store administrators, but we also need to get that information to customers. This will require some design work on our product pages, which are dependent on the storefront plugin that we’re using. Generally, this will involve making sure that our product query includes the new `volume` field and that the information is passed to the `render()` function in the file that describes our product detail page. For more information, see the [Creating and Organizing Products doc](/developer/open-commerce/docs/creating-organizing-products/).
 
+## Link multiple packages with package configuration file
```

Comment:
This really doesn't fit here--this guide is focused on the use case of writing a plugin to add a product variable and I'm not sure how this fits into that flow. Maybe it would be better in one of the plugin docs? I'm happy to get on a call and talk through what this does and figure out where it fits more specifically and then work on the wording and descriptions with respect to what a user would be doing with this information.